peregrine/acquisition.py

```diff
@@ -471,7 +483,8 @@ def progress_callback(freq_num, num_freqs):
                                      carr_freq - self.IF,
                                      code_phase,
                                      snr,
-                                     status)
+                                     status,
+                                     'l1ca')
```

Comment:
Agree. I will extend initSettings to support a list of signals to acquire: acq_signals. For now, only L1C/A will be in the list. Later, we will add Glonass there too. L2C will not be in the list, since we acquire L2C via L1C/A handover.
Acquisition instance will then be initialized with the signals from initSettings.